ee/debug/checkups/download_directory.go

```diff
@@ -0,0 +1,117 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type DownloadDirectory struct {
+	status  Status
+	summary string
+	files   []string
+}
+
+func (c *DownloadDirectory) Name() string {
+	return "Download directory contents"
+}
+
+func (c *DownloadDirectory) Run(_ context.Context, extraFH io.Writer) error {
+	downloadDir := getDownloadDir()
+	if downloadDir == "" {
+		return fmt.Errorf("no default download directory")
+	}
+
+	err := filepath.Walk(downloadDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if !info.IsDir() && isKolideInstaller(info.Name()) {
+			c.files = append(c.files, path)
+		}
+		return nil
+	})
+
+	switch {
+	case os.IsNotExist(err):
+		c.status = Warning
+		c.summary = fmt.Sprintf("download directory (%s) not present", downloadDir)
+	case err != nil:
+		c.status = Erroring
+		c.summary = fmt.Sprintf("error listing files in directory (%s): %s", downloadDir, err)
+	case len(c.files) == 0:
+		c.status = Warning
```

Comment:
I think this should be `Passing` -- `Warning` would imply a potential issue with how launcher is running.

Reply:
removed warnings